src/pages/ItemList.js

```diff
@@ -0,0 +1,43 @@
+import { db } from '../lib/firebase';
+import { useEffect, useState } from 'react';
+import { collection, getDocs, query, where } from 'firebase/firestore';
+
+function ItemList({ token }) {
+  const [docs, setDocs] = useState([]);
+
+  useEffect(() => {
+    const tokenQuery = query(
+      collection(db, 'groceries'),
+      where('user_token', '==', `${token.token}`),
```

Comment:
I'm getting an error that says `"Uncaught TypeError: Cannot read properties of null (reading 'token')"`

This is cause by `token` being undefined so we're trying to call `undefined.token`, which causes a null-pointer error. I think that we may want to refactor to make `token` just a string so that we can avoid this.